### PR TITLE
feat: add missing reactions queries methods

### PR DESCRIPTION
**Pull request by RiccardoM**
## Description
This PR adds some missing queries methods for the `x/reactions` module

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [x] targeted the correct branch (see [PR Targeting](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] followed the guidelines for [building modules](https://docs.cosmos.network/v0.44/building-modules/intro.html)
- [x] included the necessary unit and integration [tests](https://github.com/desmos-labs/desmos/blob/master/CONTRIBUTING.md#testing)
- [ ] added a changelog entry to `CHANGELOG.md`
- [x] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: proto/desmos/reactions/v1/query.proto [hunk 1]
```diff
@@ -15,7 +15,13 @@ service Query {
   // Reactions allows to query the reactions for a given post
   rpc Reactions(QueryReactionsRequest) returns (QueryReactionsResponse) {
     option (google.api.http).get =
-        "/desmos/reactions/v1/{subspace_id}/reactions";
+        "/desmos/reactions/v1/{subspace_id}/{post_id}/reactions";
```
Comment (dadamu) [inline proto/desmos/reactions/v1/query.proto:18]:
```suggestion
        "/desmos/reactions/v1/subspaces/{subspace_id}/posts/{post_id}/reactions";
```

File: proto/desmos/reactions/v1/query.proto [hunk 2]
```diff
@@ -15,7 +15,13 @@ service Query {
...
+  }
+
+  // Reaction allows to query the reaction with the given id
+  rpc Reaction(QueryReactionRequest) returns (QueryReactionResponse) {
+    option (google.api.http).get =
+        "/desmos/reactions/v1/{subspace_id}/{post_id}/reactions/{reaction_id}";
```
Comment (dadamu) [inline proto/desmos/reactions/v1/query.proto:24]:
```suggestion
        "/desmos/reactions/v1/subspaces/{subspace_id}/posts/{post_id}/reactions/{reaction_id}";
```

File: proto/desmos/reactions/v1/query.proto [hunk 3]
```diff
@@ -24,6 +30,12 @@ service Query {
     option (google.api.http).get =
         "/desmos/reactions/v1/{subspace_id}/registered-reactions";
   }
+  // RegisteredReaction allows to query the registered reaction of a subspace
+  rpc RegisteredReaction(QueryRegisteredReactionRequest)
+      returns (QueryRegisteredReactionResponse) {
+    option (google.api.http).get =
+        "/desmos/reactions/v1/{subspace_id}/registered-reactions/{reaction_id}";
```
Comment (dadamu) [inline proto/desmos/reactions/v1/query.proto:37]:
```suggestion
        "/desmos/reactions/v1/subspaces/{subspace_id}/registered-reactions/{reaction_id}";
```

File: proto/desmos/reactions/v1/query.proto [hunk 4]
```diff
@@ -24,6 +30,12 @@ service Query {
...
+  }
 
   // ReactionsParams allows to query the reaction params of a subspace
   rpc ReactionsParams(QueryReactionsParamsRequest)
```
Comment (dadamu) [inline proto/desmos/reactions/v1/query.proto:41]: as above. Should follow the restful naming rule.

**Review by dadamu — APPROVED** [review]

**Review by bragaz — APPROVED** [review]

**Pull request merged**